lib/msf/base/serializer/readable_text.rb

```diff
@@ -179,6 +179,30 @@ def self.dump_exploit_module(mod, indent = '')
     }
     output << "\n"
 
+    unless mod.side_effects.empty?
```

Comment:
@wchen-r7: Did you mean to add this code to other module types? Currently `auxiliary/gather/pimcore_creds_sqli` shows nothing.

Reply:
Additionally, this file is in dear need of a refactor.

Reply:
Additionally, this file is in dire need of a refactor. There is so much copied code.

Reply:
I just added support for basic, exploit, aux and post modules. Thanks for pointing out.